src/markdown-pages/docs/managers/github-manager.md

```diff
@@ -0,0 +1,290 @@
+```
+
+Once the GitHub issue is closed, remove the _@GitHubIssue_ annotation from the test class or test method.
+
+
+### Using Regex
+
+To ensure that the failing exception is due to the known problem that is identified in the GitHub issue, use Regex in the annotation to check that the recorded test failure matches the string that is specified in Regex. If the failing exception does not match the Regex, the test has failed for a different or unknown reason, not the reason that is specified in the GitHub issue. If no Regex is provided in the annotation, any failing exception is accepted.
+
+
+### <a name="overrides"></a>Setting overrides
+
+
+You can override the result of a test method or a test class by using the _@GitHubIssue_ annotation in conjunction with the _@ContinueOnTestFailure_ annotation to return a test result for all test methods within a test class.
+
+
+<b>Overriding a test method result:</b>
+
+If a test method is annotated with _@GitHubIssue_, and the initial result of the method is _Failed_, the GitHub Manager uses the GitHub API to retrieve the current state of the GitHub issue from the annotation. If the issue is in _open_ status, the result of the test method is overridden from _Failed_ to _Failed With Defects_. If the issue is in _closed_ status, the result is not overridden, as that issue is supposedly fixed and is not expected to affect the test method anymore. In the latter scenario, further investigation into the cause of the failure is required.
+
+<b>Overriding a test class result:</b>
+
+At the end of a test class run, the results of each individual test method are combined to determine whether the result of the test class should be overridden. 
+
+Use the following scenarios and examples to understand the expected test class result and what to do next.
+
+
+### Scenario 1: 
+
+Scenario 1 uses the _@GitHubIssue_ annotation on the test class _SimBankIVT_ without the _@ContinueOnTestFailure_ annotation:
+
+
+@Test<br>
+@GitHubIssue( issue = "1000")<br>
+Test class SimBankIVT<br>
+Test method testNotNull() result: _Passed_ ![passed icon:](passed.svg)<br>
+Test method checkBankIsAvailable() result: _Failed with defects_ ![failed with defects icon:](failed-with-defects.svg)<br><br>
+<b>Test class result: _Passed with defects_</b> ![passed with defects icon:](passed-with-defects.svg)<br>
+
+Next steps:<br>
+All methods passed, other than those expected to fail - in this case the _checkBankIsAvailable_ method. No further investigation is required.
+
+### Scenario 2: 
+
+Scenario 2 uses the _@GitHubIssue_ annotation on the test class _SimBankIVT_ with the _@ContinueOnTestFailure_ annotation:
+
+
+@Test<br>
+@GitHubIssue( issue = "1000")<br>
+@ContinueOnTestFailure<br>
+Test class SimBankIVT<br>
+Test method testNotNull() result: _Failed with defects_ ![failed with defects icon:](failed-with-defects.svg)<br>
+Test method checkBankIsAvailable() result: _Failed_ ![failed icon:](failed.svg)<br><br>
```

Comment:
how can the first method be FWD with the second is just F if the annotation is on the class?

Reply:
My mistake I overlooked the annotation being class level - I'll provide Caroline with the correct examples